stratum/hal/lib/barefoot/bf_sde_wrapper.cc

```diff
@@ -1648,7 +1648,7 @@ std::string BfSdeWrapper::GetBfChipType(int device) const {
 std::string BfSdeWrapper::GetSdeVersion() const {
 #if defined(SDE_9_3_1)
   return "9.3.1";
-#elif defined(SDE_9_5_0)
+#elif defined(SDE_9_5_0) || defined(SDE_9_5_2)
   return "9.5.0";
```

Comment:
I don't think that's smart. We should return the correct string.

Reply:
oops.
I use "search and replace" tool to change this file.
Let me fix that...